nal_eval/tests/fixtures/function.nal

```diff
@@ -0,0 +1,14 @@
+fn add_one_and_print(x) {
+    let y = x + 1
+    print(y)
+}
+
+add_one_and_print(0)
+add_one_and_print(10)
+
+fn add_one_and_return(x) {
+    return x + 1
+}
+
+print(add_one_and_return(1))
+print(add_one_and_return(11))
```

Comment:
Well, this test should be passed. It's kind of a bug. Thanks!
But due to `nal_ast::report::Report` is not yet implemented, Debugging it is just pain for now. So I'll fix it later when `Report` is ready.
But it's still valid test. You don't have to remove it.